### PR TITLE
Peoperties api

### PR DESCRIPTION
**Pull request by kylealexanderhackney**
## Link to JIRA ticket

Add a link to the JIRA ticket that the changes in this PR describe.

### *What is the problem we're trying to solve*

Add a summary of the problem and any additional context that helps explain what the issue we're trying to solve is.

### *What changes have we introduced*

Add descriptions of what changes were made to address the problem and provide a solution.

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: RepairsApi/V1/Gateways/ApiGateway.cs [hunk 1]
```diff
@@ -17,12 +18,16 @@ public ApiGateway(IHttpClientFactory clientFactory)
         }
 
         [SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Disposal of httpclient from the factory is handled by the factory")]
```
Comment (LBHBomotoso) [inline RepairsApi/V1/Gateways/ApiGateway.cs:20]: Is it better to suppress these in the editor config?

File: RepairsApi/V1/Resources.resx [hunk 2]
```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (LBHBomotoso) [inline RepairsApi/V1/Resources.resx:1]: Are these error messages that get returned on an error rather than hardcoded in the source?

**Review by LBHBomotoso — APPROVED** [review]
Looks good

File: RepairsApi/V1/Gateways/AlertsGateway.cs [hunk 3]
```diff
@@ -11,40 +13,52 @@ namespace RepairsApi.V1.Gateways
...
+            {
+                _logger.LogError($"Call to {url} failed with {response.Status}");
+                throw new PlatformApiException(response.Status);
+            }
+
             return response.Content.ToDomain();
         }
 
-        public async Task<PersonAlertList> GetPersonAlertsAsync(string tenancyReference)
+        public async Task<PersonAlertList> GetPersonAlertsAsync(string? tenancyReference)
```
Comment (LBHBomotoso) [inline RepairsApi/V1/Gateways/AlertsGateway.cs:47]: Is there a difference in declaring this reference as nullable?

Comment (kylealexanderhackney) [reply RepairsApi/V1/Gateways/AlertsGateway.cs:47]: Ive enabled nullable reference types in the domain areas so this now accurately reflects what the variable can be

File: RepairsApi/V1/Resources.resx (hunk 2 above)
Comment (kylealexanderhackney) [reply RepairsApi/V1/Resources.resx:1]: Yes Ive added some errors to the responses and the code styles dictate const string should be in a resx file for legiblity

File: RepairsApi/V1/Gateways/ApiGateway.cs (hunk 1 above)
Comment (kylealexanderhackney) [reply RepairsApi/V1/Gateways/ApiGateway.cs:20]: I would say that we should keep the error turned on. The http client and its factory or something of a special case when managing disposables. the general rule "Dispose objects before losing scope" is correct this is just an exception to that rule

File: RepairsApi/V1/Gateways/AlertsGateway.cs (hunk 3 above)
Comment (LBHBomotoso) [reply RepairsApi/V1/Gateways/AlertsGateway.cs:47]: thanks

File: RepairsApi/V1/Resources.resx (hunk 2 above)
Comment (LBHBomotoso) [reply RepairsApi/V1/Resources.resx:1]: cool

File: RepairsApi/V1/Gateways/ApiGateway.cs (hunk 1 above)
Comment (LBHBomotoso) [reply RepairsApi/V1/Gateways/ApiGateway.cs:20]: Ok, I did read something about it. https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.
Thanks

**Pull request merged**